capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'decorator'
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    super.capitalize
+  end
```

Comment:
- Hey, to make this work correctly, you're supposed to call the instance variable `@nameable` instead of super().
` @nameable.correct_name.capitalize`